download.md

```diff
@@ -3,6 +3,8 @@ layout: page
 title: Download
 ---
 
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80">](https://f-droid.org/packages/at.bitfire.davdroid)
```

Comment:
I think you should put this image in the repository, just like the other badges.